File: app/src/main/java/com/ishow/noah/modules/main/mine/MineFragment.java
package com.ishow.noah.modules.main.mine;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import com.ishow.noah.R;
import com.ishow.noah.entries.UserContainer;
import com.ishow.noah.modules.base.AppBaseFragment;

/**
 * Created by yuhaiyang on 2017/4/21.
 * Home Fragment
 */

public class MineFragment extends AppBaseFragment implements MineContract.View {


    private View mRootView;

    private MineContract.Presenter mPresenter;

    public static MineFragment newInstance() {

        Bundle args = new Bundle();

        MineFragment fragment = new MineFragment();
        fragment.setArguments(args);
        return fragment;
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        if (mRootView != null) {
            return mRootView;
        }

        mRootView = inflater.inflate(R.layout.fragement_mine, container, false);

        mPresenter = new MinePresenter(this);
        return mRootView;
    }

    @Override
    public void onResume() {
        super.onResume();
        mPresenter.onResume(getContext());
    }


    @Override
    public void update(UserContainer userContainer) {
        /*
        ImageLoader.with(getContext())
                .load(userContainer.getUser().getAvatar())
                .mode(ImageLoader.LoaderMode.CIRCLE_CROP)
                .placeholder(R.drawable.logo)
                .into(mAvatarView);


        mNameView.setText(userContainer.getUser().getNickName());
        */
    }
}

File: app/src/main/java/com/ishow/noah/modules/main/mine/MineFragment.kt
package com.ishow.noah.modules.main.mine

import android.os.Bundle
import android.view.View
import com.ishow.noah.R
import com.ishow.noah.databinding.FMineBinding
import com.ishow.noah.modules.base.mvvm.view.AppBindFragment

/**
 * Created by yuhaiyang on 2020-05-11.
 */
class MineFragment : AppBindFragment<FMineBinding, MineViewModel>() {

    override fun getLayout(): Int = R.layout.f_mine


    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
    }

    companion object {
        fun newInstance(): MineFragment {

            val args = Bundle()

            val fragment = MineFragment()
            fragment.arguments = args
            return fragment
        }
    }
}